src/_transforms/parse.js

```diff
@@ -1,7 +1,7 @@
 import { parseHTML } from "linkedom";
 
 export default (value, outputPath) => {
-    if (outputPath && outputPath.includes(".html")) {
+    if (outputPath && outputPath.includes(".html") && outputPath.includes("about")) {
```

Comment:
@greatislander what do you think about this approach to only run this parser in about pages?

Reply:
I think the better approach if you want to restrict where the parser is run is to add a class to the `<main>` element of pages where the parser should wrap content in sections, e.g. `.sectioned`. Then you can modify the query below:

```javascript
const sectionHeadings = document.querySelectorAll("main.sectioned h2:not([class])");
```

Your approach here would necessitate modifying the parser every time a new page required this functionality; it's more robust to be able to add support to pages whenever needed based on a class.